src/server/database/Updater/DBUpdater.cpp

```diff
@@ -178,7 +178,7 @@ bool DBUpdater<T>::Create(DatabaseWorkerPool<T>& pool)
     // Path of temp file
     static Path const temp("create_table.sql");
 
-    // Create temporary query to use external mysql cli
+    // Create temporary query to use external MySQL CLi
```

Comment:
Fair point, I just read it as "Command Line interface" when I see it spelled `CLi` elsewhere. :-)
It's a moot point now that this PR is merged. I wasn't quick enough to read your comment. ;)